CHANGELOG.md

```diff
@@ -4,14 +4,53 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.4.0 (2021-07-30)
+
+This release adds new experimental features since the 2.3.1 release.
+
+The experimental features in this release are:
+* APIs for chunk manipulation across data nodes in a distributed
+hypertable setup. This includes the ability to add a data node and move
+chunks to the new data node for cluster rebalancing.
+* The `time_bucket_ng` function, a newer version of `time_bucket`. This 
+function supports years, months, days, hours, minutes, and seconds.
+ 
+We’re committed to developing these experiments, giving the community
+ a chance to provide early feedback and influence the direction of
+TimescaleDB’s development. We’ll travel faster with your input! 
+	
+Please create your feedback as a GitHub issue (using the 
+experimental-schema label), describe what you found, and tell us the 
+steps or share the code snip to recreate it.
+
+This release also includes several bug fixes. 
+
+PostgreSQL 11 deprecation announcement
+Timescale is working hard on our next exciting features. To make that 
+possible, we require functionality that is available in Postgres 12 and 
+above. For this reason, from TimescaleDB 2.4 onwards PostgreSQL 12 or 
+13 are required.
```

Comment:
Can we be more explicit here and say PostgreSQL 11 is not supported?

Reply:
@Loquacity?
```suggestion
13 are required. PostgreSQL 11 is not supported on TimescaleDB 2.4.
```

Reply:
changed the wording to ^.